### PR TITLE
#22 Adds key word glossary

### PR DESCRIPTION
**Pull request by Cleop**
#22 Adds key word glossary
#15 Adds a link to the user testing repo
#13 definition of a milestone

### CODE REVIEW

**Review by nelsonic — APPROVED** [review]
👀 👍 🎉

File: readme.md [hunk 1]
```diff
@@ -111,6 +111,34 @@ cuts out back and forth of correction had someone else written it.
...
+- **Scrum Master**: tracks team progress, liaises with product owner on
+requirements, lends expertise on managing end to end projects, ensures features
+are tested when delivered and manages deployments in conjunction with technical
+architect and the team.
+- **Product backlog:** a prioritised features / to do list for the product found
+on github. See our ['What is the backlog?' section](#what-is-the-backlog?) for
+more detail.
+- **Sprint Backlog**: the backlog that is aiming for completion within a given
+sprint period.
+- **Sprint**: a 1-3 week working lifecycle of planning, building, presentation and
```
Comment (iteles) [inline readme.md:125]: I would probably move `Sprint` up above the backlogs (as you refer to a sprint backlog before defining a sprint) but not a blocker!

File: readme.md [hunk 2]
```diff
@@ -111,6 +111,34 @@ cuts out back and forth of correction had someone else written it.
...
+reflection. We tend to work in 2 week (10 working day) sprints.
+- **UI / User Interface**: how the site looks from a design perspective.
+- **User testing:** interviews, focus groups and other forms of research conducted
```
Comment (iteles) [inline readme.md:128]: We probably want to add `user research` and `user journey` to the glossary above this too.

File: readme.md [hunk 3]
```diff
@@ -111,6 +111,34 @@ cuts out back and forth of correction had someone else written it.
...
+with potential/actual users of your product to give feedback and provide insight
+into what the user's needs are. See
+https://github.com/dwyl/learn-user-experience-testing to learn more about user
+testing.
+- **Milestone**:  a visual way to group issues in github, at dwyl we create one
+milestone per sprint so we have somewhere to see all of the issues which we
+estimate to complete by the end of the sprint. See
+https://github.com/dwyl/github-reference for more.
```
Comment (iteles) [inline readme.md:136]: There's apparently nothing on the github-reference repo on milestones!!!

**Review by iteles — APPROVED** [review]
@Cleop Left some comments, but no blockers 👍

File: readme.md (hunk 3 above)
Comment (Cleop) [reply readme.md:136]: There's just an issue: https://github.com/dwyl/github-reference/issues/13

**Pull request merged**